core/ordering/cosipbft/mod.go

```diff
@@ -465,109 +476,128 @@ func (s *Service) doRound(ctx context.Context) error {
 		return xerrors.Errorf("reading roster: %v", err)
 	}
 
+	timeout := s.timeoutRound
+	if s.failedRound {
+		timeout = s.timeoutRoundAfterFailure
+	}
+
 	leader, err := s.pbftsm.GetLeader()
 	if err != nil {
 		return xerrors.Errorf("reading leader: %v", err)
 	}
 
-	timeout := s.timeoutRound
-	if s.failedRound {
-		timeout = s.timeoutRoundAfterFailure
+	if s.me.Equal(leader) {
+		s.logger.Info().Msgf("doLeaderRound with timeout %f", float32(timeout)/1000000000)
```

Comment:
why go through such a complicated expression ? 
Also, shouldn't this be a debug message ? 
```suggestion
		s.logger.Debug().Msgf("doLeaderRound with timeout %s", timeout)
```

Maybe the message could be improved too, as this is not very human friendly.